src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -60,6 +61,8 @@ const getStoredElements = () => {
     const editorData = JSON.parse(localStorage.getItem(OATDataStorageKey));
     return editorData && editorData.models ? editorData.models : null;
 };
+const nodeWidth = 300;
+const nodeHeight = 100;
```

Comment:
Are these values present anywhere else in the code base?

Reply:
No those ones are average sizes for the nodes, for the floating edges we have a way to measure the exact size of each independent node, but for layouting matters that effort don't seem necessary.